.github/workflows/auto-label.yml

```
@@ -0,0 +1,70 @@
+    permissions:
+      contents: read
+      pull-requests: write
+      issues: write
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Auto label based on file changes
+        if: github.event_name == 'pull_request'
+        uses: actions/labeler@v4
```

actionlint (1.7.7)

the runner of "actions/labeler@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)